### ISSUE TITLE
Corrections to Anthology IDs 2021.naacl-main.64, 2021.naacl-main.343, 2021.naacl-main.398

### ISSUE THREAD

**Issue by rspeer** [acl-org/acl-anthology#1331]
## Metadata correction: please describe the issue here**

My name is Robyn Speer, and I'm a trans author who has requested this kind of revision before. These three papers in NAACL 2021 cite me by my deadname:

* 2021.naacl-main.64
* 2021.naacl-main.343
* 2021.naacl-main.398

I'm opening this issue as soon as I have the information, because we are in a race against Google Scholar indexing the papers.

NAACL announced a process for preventing deadnamed citations on May 13, in this post: https://2021.naacl.org/blog/name-change-procedure/. I made a request under that process on May 16. Yesterday, I learned that NAACL had failed to implement the process, the proceedings would be released today unchanged, and that NAACL 2021 included papers that cited me by my deadname.

It is very important to me to prevent new citations of my deadname from spreading, especially to Google Scholar. (Google Scholar has no accountability, and the only way to correct your name in it is to feed their algorithm with a sufficient amount of correct data over time.)

## Revision or erratum: please add the following information**

- [ ] I have attached the revised PDF or erratum to this issue
- [X] I have written a brief description of the changes below

I am not the author of any of these papers; I am the author of a paper they cite. I cannot provide a revised PDF. I can only let you know what needs to be revised, which is my name.

**Description of changes**

The revision should change my deadname (a different name that could be abbreviated R. Speer) to "Robyn Speer". Because my deadname is harmful personal information, no description of the change should be made on the revision.

**Comment by mjpost** [comment]
Hi Robyn—Have you contacted the authors about these papers? We can expedite a revision if we have their consent, and we can use an edited version of their PDF to save them the work. A comment here or an email from them would suffice.

**Comment by a3616001** [comment]
Hi Robyn,

I am sorry this happened. 

I am good with this revision on [our NAACL paper](https://www.aclweb.org/anthology/2021.naacl-main.398/).


Best,
Zexuan

**Comment by JerichoWorld** [comment]
Hi Robyn,

I'm the first author on 2021.naacl-main.64. I deeply apologize for using this citation (I was not aware, I should have checked outside Google Scholar). I consent to changing the citation, and have already submitted an update fixing the citation to the ArXiv version as well. Let me know if I need to do anything else.

--Raj

**Comment by rajammanabrolu** [comment]
Hi Robyn,

I'm the first author on 2021.naacl-main.64. I deeply apologize for using this citation (I was not aware, I should have checked outside Google Scholar). I consent to changing the citation, and have already submitted an update fixing the citation to the ArXiv version as well. Let me know if I need to do anything else.

--Raj

**Comment by rspeer** [comment]
Thanks for all of this so far.

One thing I should make sure to mention, just because I don't know if it'll naturally happen: when these papers are assigned DOIs, make sure the DOI points to the v2 paper.

**Comment by pranav-ust** [comment]
Hey @mjpost, I am the D&I chair for NAACL. May I know what are the next steps regarding this and when the citations would be fixed? 

Also, I am quite surprised to see that authors' consent is being asked to correct the deadnames in the citations. At least for fixing the deadnames, I don't think that authors' consent should be required. What if the authors' never reply and the paper with deadnames get circulated publicly?

**Comment by akoehn** [comment]
> One thing I should make sure to mention, just because I don't know if it'll naturally happen: when these papers are assigned DOIs, make sure the DOI points to the v2 paper.

The DOIs point to the html, not to the pdf. Therefore, it will always link to the latest version.

@pranav-ust we discussed this internally and came to the conclusion that we as the anthology volunteers cannot just alter the papers of a conference without consent. This kind of check and change would need to happen on the conference site.

As to the next steps: I already prepared corrected PDFs yesterday and they will go live once #1330 is merged.
